helusers/jwt.py

```diff
@@ -79,10 +81,15 @@ def validate_api_scope(self):
         if not self.settings.REQUIRE_API_SCOPE_FOR_AUTHENTICATION:
             return
 
-        api_scope = self.settings.API_SCOPE_PREFIX
-        if not self.has_api_scope_with_prefix(api_scope):
+        api_scopes = api_token_auth_settings.API_SCOPE_PREFIX
```

Comment:
Should use `self.settings` instead of `api_token_auth_settings`?

Reply:
Yeah. Good catch!